website/docs/getting-started-herotypes.md

```diff
@@ -0,0 +1,148 @@
+
+Example of a function that takes and returns a _TextSeries_:
+```python
+>>> s = pd.Series(["Text: of first! document", "Text of second ... document"])
+>>> hero.clean(s)
+0     text first document
+1    text second document
+dtype: object
+```
+
+<h3 align="left">TokenSeries</h3>
+
+In a _TokenSeries_, every cell is a list of words/tokens. We use this to prepare our data for _representation_, so to gain insights from it through mathematical methods. This is why the functions that initially transform your documents to vectors, namely `tfidf, term_frequency, count`, take a _TokenSeries_ as input.
+
+Example of a function that takes a _TextSeries_ and returns a _TokenSeries_:
+```python
+>>> s = pd.Series(["text first document", "text second document"])
+>>> hero.tokenize(s)
+0     [text, first, document]
+1    [text, second, document]
+dtype: object
+```
+
+<h3 align="left">VectorSeries</h3>
+
+In a _VectorSeries_, every cell is a vector representing text. We use this when we have a low-dimensional (e.g. vectors with length <=1000), dense (so not a lot of zeroes) representation of our texts that we want to work on. For example, the dimensionality reduction functions `pca, nmf, tsne` all take a high-dimensional representation of our text (in the form of a _DataFrame_ (see below) or _VectorSeries_, and return a low-dimensional representation of our text in the form of a _VectorSeries_.
+
+Example of a function that takes as input a _DataFrame_ or _VectorSeries_ and returns a _VectorSeries_:
+```python
+>>> s = pd.Series(["text first document", "text second document"]).pipe(hero.tokenize).pipe(hero.term_frequency)
+>>> hero.pca(s)
+0     [0.118, 0.0]
+1    [-0.118, 0.0]
+dtype: object
+```
+
+<h3 align="left">DataFrame</h3>
+
+In Natural Language Processing, we are often working with matrices that contain information about our dataset. For example, the output of the functions `tfidf`, `count`, and `term_frequency` is a [Document Term Matrix](https://en.wikipedia.org/wiki/Document-term_matrix), i.e. a matrix where each row is one document and each column is one term / word.
+
+We use a Pandas DataFrame for this for two reasons:
+1. It looks nice.
+2. It can be sparse.
+
+The second reason is worth explaining in more detail: In e.g. a big Document Term Matrix, we might have 10,000 different terms, so 10,000 columns in our DataFrame. Additionally, most documents will only contain a small subset of all the terms. Thus, in each row, there will be lots of zeros in our matrix. This is why we use a [sparse matrix](https://en.wikipedia.org/wiki/Sparse_matrix): A sparse matrix only stores the non-zero fields. And Pandas DataFrames support sparse data, so Texthero users fully profit from the sparseness!
```

Comment:
Document-term and every where else as well

Reply:
will do :paintbrush: